helper-intro.md

```diff
@@ -1,80 +1,53 @@
+Most importantly, *you do not have to know everything* (we don't, either), but you are expected to:
 
-- Prerequisites: Have been to a CodeRefinery before and used git some
-  since then, or have a bit more some experience with git (branching,
-  pull request, github) and command line work.  Be able to focus
-  during the course.
+- Have been to a CodeRefinery before and used git some since then, OR have some general experience with git (branching,
+  pull requests) and command line work.
 
-- *You do not have to know everything*, but keep exercises going and
-   call us when there are difficult questions!
+- Be present in your assigned breakout room during exercise sessions of the workshop.
 
+- Show a positive, motivating attitude to learners.
 
-## Helper training
+- Keep exercises going and let us know when there are difficult questions!
 
-During this time, we will talk about the role of the helper during the
-workshop and answer any open questions you may have.  We'll go through
-the material below.
+- Come to a one-hour "Exercise leader onboarding" (see below) before the workshop.
 
-If you have any doubts, questions, ideas or anything you want to tell
-us apart from these sessions, please ask!  We are here to help.  Our
-general email is <support@coderefinery.org> or you can join the
-{doc}`CodeRefinery chat <chat>` (we recommend the `#workshops` stream,
-and if you can't find it then `#general` is good).
 
+## Exercise leader onboarding
 
-## Reading
+We usually offer two one-hour timeslots in the week before the workshop (check the schedule!) for exercise leader onboarding. You are expected to join one of these times. During this time, we will talk about the role of the exercise leader during the workshop and answer any open questions you may have. We will also go through the material below.
 
-We have these things we recommend you read in advance (don't worry,
-we'll go over them in helper training):
+## Material for exercise leaders
 
-- [Basics of being a good, motivating helper](helping-and-teaching)
-- {ref}`How breakout rooms work <breakout-helpers>`, focused on
-  sections for helpers.
+You can already read through the material below, if you have time, but do not worry, we will be going over it in the exercise leader onboarding session.
 
+- [Basics of being a good, motivating exercise leader](helping-and-teaching)
+- [How breakout rooms work](breakout-rooms-helping), check the section for exercise leaders.
 
-You'll learn these during the workshop so *you don't need to read
-these in advance* (and if you've been to a CodeRefinery before, you'll
-know this):
+If you have been to a CodeRefinery workshop before, you will most likely know the following, but if unsure, also check the links below: 
 
 - [HackMD mechanics](hackmd-mechanics)
 - [Zoom mechanics](zoom-mechanics)
-- It is not bad to read hints on [how to attend
+- It also is not bad to read hints on [how to attend
```

Comment:
I am nothing native English speaker, but in the later part "also" is used again, so probably not necessary here?

Reply:
will remove